diff --git a/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java b/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java
index 094b395..30c7915 100644
--- a/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java
+++ b/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java
@@ -21,7 +21,9 @@ public class HTMLUtil {
         source = source.replaceAll("(?!<(img).*?>)<.*?>", "")
                 .replaceAll("(onload(.*?)=)", "")
                 .replaceAll("(onerror(.*?)=)", "");
-        return deleteHMTLTag(source);
+        source = deleteHMTLTag(source);
+        source = sensitiveWordBs.replace(source);
+        return source;
     }
 
     public static String deleteHMTLTag(String source) {
