diff --git a/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java b/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java
index 094b395..265f58a 100644
--- a/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java
+++ b/aurora-springboot/src/main/java/com/aurora/util/HTMLUtil.java
@@ -4,24 +4,13 @@ import com.github.houbb.sensitive.word.bs.SensitiveWordBs;
 
 public class HTMLUtil {
 
-    // temporary unused
-    private static final SensitiveWordBs sensitiveWordBs = SensitiveWordBs.newInstance()
-            .ignoreCase(true)
-            .ignoreWidth(true)
-            .ignoreNumStyle(true)
-            .ignoreChineseStyle(true)
-            .ignoreEnglishStyle(true)
-            .ignoreRepeat(true)
-            .enableNumCheck(false)
-            .enableEmailCheck(false)
-            .enableUrlCheck(false)
-            .init();
+    private static final SensitiveWordBs sensitiveWordBs = SensitiveWordBs.newInstance().init();
 
     public static String filter(String source) {
         source = source.replaceAll("(?!<(img).*?>)<.*?>", "")
                 .replaceAll("(onload(.*?)=)", "")
                 .replaceAll("(onerror(.*?)=)", "");
-        return deleteHMTLTag(source);
+        return sensitiveWordBs.replace(deleteHMTLTag(source));
     }
 
     public static String deleteHMTLTag(String source) {
